### PR TITLE
Completed javascript-conditionals assignment

### PR DESCRIPTION
**Pull request by Aiden-Peace300**
image
image

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+  age: 39, // a interger is being assigned to the age property
+}; // closing code block
+
+// the log method of the console object called with two arguments a string and 'isOldEnoughToDrive()' function with 'homer' object as a single arguement
+console.log('isOldEnoughToDrive(homer):', isOldEnoughToDrive(homer));
+// true
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age >= 16 && person.age >= 21) {
+    return true; // returning the boolean data type 'true'
```
Comment (shawnkost) [inline javascript-conditionals/main.js:128]: You should never be able to drink and drive!

File: javascript-conditionals/main.js [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+// the log method of the console object called with two arguments a string and 'isOldEnoughToDrinkAndDrive()' function with 'homer' object as a single arguement
+console.log(
+  'isOldEnoughToDrinkAndDrive(lisa):',
+  isOldEnoughToDrinkAndDrive(lisa)
+);
+// false
+
+function categorizeAcidity(pH) {
+  if (pH >= 0 && pH <= 14) {
+    if (pH >= 0 && pH < 7) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:150]: Works, but the nested `if` statements isn't necessary. You're already checking if it's greater than 0, or less than 14 in another statements inside of here, so you don't need the nested.

File: javascript-conditionals/main.js [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+      return 'acid';
+    } else if (pH === 7) {
+      return 'neutral';
+    } else if (pH >= 7 && pH < 14) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:154]: This should just be `> 7` && `<= 14`.

File: javascript-conditionals/main.js [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+    case 'comedy':
+      return 'The Big Lebowski';
+    case 'horror':
+      return 'Psycho';
+    case 'drama':
+      return 'Edward Scissorhands';
+    case 'musical':
+      return 'Highschool Musical';
+    default:
+      return 'Genre not recognized. Choose between action, comedy, horror, drama, musical, or sci-fi';
```
Comment (shawnkost) [inline javascript-conditionals/main.js:226]: Missing 1 more `case` clause for `sci-fi`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Aiden! A few tweaks here

**Comment by Aiden-Peace300** [comment]
image
image

File: javascript-conditionals/main.js [hunk 5]
```diff
@@ -146,14 +141,12 @@ console.log(
...
-    } else if (pH === 7) {
-      return 'neutral';
-    } else if (pH >= 7 && pH < 14) {
-      return 'base';
-    }
+  if (pH >= 0 && pH < 7) {
+    return 'acid';
+  } else if (pH === 7) {
+    return 'neutral';
+  } else if (pH > 7 && pH < 14) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:148]: What happens when `pH` is 14? 14 is still considered a base

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change here Aiden!

**Comment by Aiden-Peace300** [comment]
image
image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**